### PR TITLE
Temporary fix for module tracer by disabling rewrite

### PR DESCRIPTION
**Pull request by cblappert**
Added regression test and re-enabled resolve initialized modules.

Also, fixed bug in cli.

### CODE REVIEW

File: src/prepack-cli.js [hunk 1]
```diff
@@ -172,13 +172,15 @@ function run(
     }
     if (errors.size > 0) {
       console.log("Errors found while prepacking");
+      let foundFatal = false;
       for (let [loc, error] of errors) {
+        foundFatal = foundFatal || error.severity === "FatalError";
```
Comment (NTillmann) [inline src/prepack-cli.js]: Look 20 lines up --- same code. Probably should get same treatment.

File: src/serializer/modules.js [hunk 2]
```diff
@@ -240,9 +240,9 @@ class ModuleTracer extends Tracer {
               this.nestedRequiredModulesAndValues.push({ id: moduleIdValue, value: result });
             }
             if (isTopLevelRequire) {
-              for (let { id, value } of this.nestedRequiredModulesAndValues) {
+              /*for (let { id, value } of this.nestedRequiredModulesAndValues) {
                 this.modules.initializedModules.set(id, value);
-              }
+              }*/
```
Comment (hermanventer) [inline src/serializer/modules.js]: What's with the commented out code?

Comment (cblappert) [inline src/serializer/modules.js]: Its the new code that figures out when modules are initialized or not, but it doesn't quite work, I'll uncomment it in the next PR.

**Comment by cblappert** [comment]
CircleCI is failing on test-serializer-with-coverage with a stack size exceeded error (but it passes test-serializer). We need to figure out a way to increase the stack size for that test (or all tests).

**Pull request closed without merging**